File: LABORATORIO_01/RegistroEstudiante2/app/src/main/java/com/example/registroestudiante/MainActivity.java
package com.example.registroestudiante;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText nombre, apellidos, edad;
    Button registrar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        nombre = findViewById(R.id.etNombre);
        apellidos = findViewById(R.id.etApellidos);
        edad = findViewById(R.id.etEdad);
        registrar = findViewById(R.id.btnRegistrar);

        registrar.setOnClickListener(v -> {

            String n = nombre.getText().toString();
            String a = apellidos.getText().toString();
            String e = edad.getText().toString();

            Toast.makeText(this,
                    "Estudiante: " + n + " " + a + " Edad: " + e,
                    Toast.LENGTH_LONG).show();
        });
    }
}

File: LABORATORIO_01/RegistroEstudianteK/app/src/main/java/com/example/registroestudiantek/MainActivity.kt
package com.example.registroestudiantek

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val nombre = findViewById<EditText>(R.id.etNombre)
        val apellidos = findViewById<EditText>(R.id.etApellidos)
        val edad = findViewById<EditText>(R.id.etEdad)
        val registrar = findViewById<Button>(R.id.btnRegistrar)

        registrar.setOnClickListener {

            val n = nombre.text.toString()
            val a = apellidos.text.toString()
            val e = edad.text.toString()

            Toast.makeText(
                this,
                "Estudiante: $n $a Edad: $e",
                Toast.LENGTH_LONG
            ).show()
        }
    }
}